code/modules/mob/living/carbon/carbon.dm

```diff
@@ -781,5 +781,12 @@
 
 	..()
 
-
+/mob/living/carbon/adjustToxLoss(amount, updating_health=1)
+	if(TOXINLOVER in dna.species.specflags) //damage becomes healing and healing becomes damage

```

Comment:
carbon species have var/datum/dna/dna = null
Humans have dna, and monkeys have dna
Aliens and "true demons" might not. I guess I should check all the same in this situation.